doc/start/documenting-ceph.rst

```diff
@@ -326,7 +326,7 @@ distributions, execute the following:
 
 .. prompt:: bash $
 
-	sudo apt-get install gcc python-dev python-pip libxml2-dev libxslt-dev doxygen graphviz ant ditaa
+	sudo apt-get install gcc python-dev python3-pip libxml2-dev libxslt-dev doxygen graphviz ant ditaa
```

Comment:
i see a lot of them can be changed to python3-devel python3-pip  python3-dev if you have the bandwidth, want to updaye them as well?